Description: temporary patch to lower kolabxml dep
 It was bumped upstream in a guestimate patch but given Apps15.08 built with
 kolabxml 1.1 chances are master will as well, so this patch is meant to 
 test that hypothesis
 https://quickgit.kde.org/?p=kdepim-runtime.git&a=commit&h=6f396740e8b083ef7470466121c61a7372ecd55f
Author: Harald Sitter <sitter@kde.org>
Origin: vendor
Forwarded: no
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: kdepim-runtime/resources/CMakeLists.txt
===================================================================
--- kdepim-runtime.orig/resources/CMakeLists.txt
+++ kdepim-runtime/resources/CMakeLists.txt
@@ -21,7 +21,7 @@ find_package(Libkolab 1.0 QUIET CONFIG)
 set_package_properties(Libkolab PROPERTIES DESCRIPTION "libkolab" URL "http://mirror.kolabsys.com/pub/releases" TYPE OPTIONAL PURPOSE "The Kolab Format libraries are required to build the Kolab Groupware Resource")
 
 # Libkolabxml
-find_package(Libkolabxml 1.2 QUIET CONFIG)
+find_package(Libkolabxml 1.1 QUIET CONFIG)
 set_package_properties(Libkolabxml PROPERTIES DESCRIPTION "Kolabxml" URL "http://mirror.kolabsys.com/pub/releases" TYPE OPTIONAL PURPOSE "The Kolab XML Format Schema Definitions Library is required to build the Kolab Groupware Resource")
 
 include_directories(
